apps/src/templates/watchers/Watchers.jsx

```diff
@@ -462,3 +475,26 @@ function nonValueDescriptor(obj) {
 function wrapValue(index, length) {
   return (index + length) % length;
 }
+
+function parseArray(array) {
+  let parsedArray = '';
+  array.forEach((element, index, array) => {
+    if (element === null) {
+      parsedArray = parsedArray + 'null';
```

Comment:
nit: += would be more concise

Reply:
Thanks! Was still thinking in python from teaching in the AM. :)